sdk/maps/azure-maps-traffic/src/main/java/com/azure/maps/traffic/models/TrafficIncidentViewportOptions.java

```diff
@@ -0,0 +1,126 @@
+package com.azure.maps.traffic.models;
+
+import com.azure.core.models.GeoBoundingBox;
+
+public final class TrafficIncidentViewportOptions {
+    private ResponseFormat format;
```

Comment:
make private / out of options

Reply:
To clarify does this comment mean to move ResponseFormat from public into private model and as a result import ResponseFormat from implementation.models across all options classes?